appyters/RNA_Level_Functional_Predictions/templates/fields/AutocompleteField.j2

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<style>
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font: 16px Arial;  
+  margin-top: 10%;
+  text-align: center;
+}
+.autocomplete {
+  position: relative;
+  display: inline-block;
+}
+
+input {
```

Comment:
This would affect other inputs as well, using this field effectively changes the styling of the whole page